gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/generator/BundleGenerator.java

```diff
@@ -0,0 +1,34 @@
+package uk.nhs.adaptors.pss.translator.generator;
+
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.UriType;
+
+import org.springframework.stereotype.Component;
+
+import uk.nhs.adaptors.pss.translator.service.FhirIDGeneratorService;
+
+@Component
+public class BundleGenerator {
```

Comment:
You can add `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` annotation and remove the constructor.